astropy/convolution/tests/test_convolve.py

```diff
@@ -944,7 +944,7 @@ def test_uninterpolated_nan_regions(boundary, normalize_kernel):
     image = np.pad(nan_centroid, pad_width=kernel.shape[0]*2, mode='constant',
                    constant_values=1)
     with pytest.warns(AstropyUserWarning,
-                      match="nan_treatment='interpolate', however, NaN values detected "
+                      match=r"nan_treatment='interpolate', however, NaN values detected "
                       "post convolution. A contiguous region of NaN values, larger "
```

Comment:
Needs the `r` in front of the quotes on **every line** that should be interpreted as raw-string.

Reply:
Sorry for missing that. Something did look wrong about the multi-line strings, probably was too late for semi-automated mass-refactoring...